gadget/gadget-amd64.yaml

```diff
@@ -1,3 +1,7 @@
+defaults:
+  system:
+    service.ssh.disable: false
```

Comment:
By default it is enabled, unless I am missing something here, so this is not necessary.